src/components/RegistrationForm.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import HomeNav from './HomeNav'
+import { Router, Route, Link } from 'react-router'
+
+
+export default class RegistrationForm extends Component {
+  constructor( props) {
+    super ( props )
+    this.state = {fullName: '', email: '', password: '', phoneNumber: '', company: ''}
+
+  }
+
+  componentDidMount() {
+    const postIsHappening = {
+      method: 'POST', mode: 'cors', headers: new Headers({
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+      })
+    }
+    const postString = `http:localhost:5000/api/v1/users`
+    fetch( postString, postIsHappening )
+    .then( data => data.json())
+  }
+
+  handleName( event ) {
+    this.setState({ name: event.target.fullName })
+  }
+
+  handleEmail( event ) {
+    this.setState({ email: event.target.email })
+  }
+
+  handlePass( event ) {
+    this.setState({ password: event.target.password })
+  }
+
+  handleNumber( event ) {
+    this.setState({ phone_number: event.target.number })
+  }
+
+  handleCompany( event ) {
+    this.setState({ company: event.target.company })
+  }
+
+  handleChange( event ) {
+    this.setState({ newsletter_subscribed: true })
+  }
+
+  handleSubmit( event ) {
+    console.log('successfully submitted.')
+  }
+
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit} action="http://localhost:5000/api/v1/users/register" method="post">
+        <input
+          type="text"
+          name="name"
+          placeholder="Name"
+          fullName={this.state.fullName}
```

Comment:
This should be value:{this.state.fullName}, not fullName. The browser is throwing lots of errors. Check the console errors!